include/openssl/stack.h

```diff
@@ -614,10 +635,10 @@ inline
     typename std::enable_if<!internal::StackTraits<Stack>::kIsConst, bool>::type
     PushToStack(Stack *sk,
                 UniquePtr<typename internal::StackTraits<Stack>::Type> elem) {
-  if (!sk_push(reinterpret_cast<_STACK *>(sk), elem.get())) {
+  if (!OPENSSL_sk_push(reinterpret_cast<_STACK *>(sk), elem.get())) {
```

Comment:
Should this be:
```suggestion
  if (!OPENSSL_sk_push(reinterpret_cast<OPENSSL_STACK *>(sk), elem.get())) {
```
?